internal/cmd/test.go

```diff
@@ -83,6 +95,44 @@ func getUnusedPort() (int, error) {
 	return listener.Addr().(*net.TCPAddr).Port, nil
 }
 
+func MustStartCMDServer() {
+	port, err := getUnusedPort()
+	if err != nil {
+		log.Fatal("Failed to get an unused port.", err)
+	}
+
+	old_port = config.WEB_PORT.Get()
+	config.WEB_PORT.Set(port)
+
+	var serverRun sync.WaitGroup
+	serverRun.Add(1)
+
+	go func() {
+		serverRun.Done()
+
+		err = procedures.Start()
+		if err != nil {
+			log.Fatal("Failed to start the server.", err)
+		}
+	}()
+
+	// Small sleep to allow the server to start up.
+	time.Sleep(100 * time.Millisecond)
+}
+
+func MustStopCMDServer() {
```

Comment:
What makes this a `Must` function? It looks like we can log errors in `server.StopServer()`, but this cannot return errors (so it should never panic).

I think you want to mirror the function name of `MustStartCMDServer()` (which should be a `Must` because it can Fatal / panic). But, we probably shouldn't add the `Must` to `StopCMDServer()` since other people might get scared to call this code if they do not want to panic.